npm/create-cypress-tests/package.json

```diff
@@ -5,7 +5,8 @@
   "private": false,
   "main": "index.js",
   "scripts": {
-    "build": "yarn prepare-example && tsc -p ./tsconfig.json && node scripts/example copy-to ./dist/initial-template && yarn copy \"./src/**/*.template.js\" \"./dist/src\"",
+    "build": "yarn prepare-example && tsc -p ./tsconfig.json && node scripts/example copy-to ./dist/initial-template && yarn copy-templates",
```

Comment:
@elevatebart @Manuel-Suarez-Abascal what do this do? Seems CI is not passing after this PR - I wonder what changed to make it fail. This seems like the only real change other than dependencies.

Reply:
I had some trouble with yarn and copy package on my laptop, done with @elevatebart